village/src/App.js

```diff
@@ -14,11 +17,28 @@ class App extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
+  
+  componentDidMount(){
+    axios.get('http://localhost:3333/smurf')
+    .then((response) => {
+      this.setState(() =>({smurfs: response.data}))
+    })
+    .catch(err=>console.log(err))
+  }
+
+
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+      <div className ='nav'>
+          <NavLink className = 'nav-item'to='/'>Home</NavLink>
+          <NavLink className = 'nav-item'to='/smurf-form'>Create Smurf</NavLink>
+      </div>
+      <Route exact path="/"
+      render={props =>(
+        <smurfForm {...props}/>
```

Comment:
check your console for errors, there is something in there that might give you a hint as to why this is not rendering